commons-data-impl/src/main/java/com/appjars/saturn/dao/jpa/ConversionJpaDaoSupport.java

```diff
@@ -72,13 +76,31 @@ default K save(S entity) {
 	@Override
 	default void saveOrUpdate(S entity) {
```

Comment:
With this implementation, my understanding is that basically what this method does is to "update" the entity (it won't "save" a new entity). So maybe we should consider change its name from `saveOrUpdate` to just `update`.

Reply:
Done ✔